services/workers/src/main/java/org/sagebionetworks/file/worker/S3FileCopyWorker.java

```diff
@@ -295,7 +295,7 @@ private long precheckFile(Entry entry, UserInfo userInfo, boolean overwrite, Str
 				throw new IllegalArgumentException(
 						"The bucket "
 								+ entry.getFileCopyResult().getResultBucket()
-								+ " could not be found or accessed. Check the name and permissions on that bucket. See www.synapse.org//#!HelpPages:CreatingADownloadBucket for details on how to set up a download bucket.");
+								+ " could not be found or accessed. Check the name and permissions on that bucket. See https://www.synapse.org//#!Help:CreateS3DownloadBucket for details on how to set up a download bucket.");

```

Comment:
Could you remove the extra forward-slash '/'?  Any way you could include fixing the typo for https://sagebionetworks.jira.com/browse/PLFM-3463 in here too?